quantus/helpers/enums.py

```diff
@@ -0,0 +1,61 @@
+from enum import Enum
+
+
+class DataType(Enum):
+    """
+    This enum represents the different types of data that a metric implementation currently supports.
+
+        - IMAGE: Represents image data.
+        - TABULAR: Represents tabular data.
+        - TEXT: Represents text data.
+    """
+
+    IMAGE = "image"  # 3D data
+    TIMESERIES = "time-series"  # 1D data
+    TABULAR = "tabular"  # 2D data
+    TEXT = "text"
+
+
+class ModelType(Enum):
```

Comment:
I'd say `torch` and `tensorflow` describe the underlying framework used for tensor computation, and `ModelType` sound like it is about architecture or task it is solving, so this one is a bit confusing. It would also be great to have some sort of list of supported DNN libraries, since they all have slightly different API, e.g., `torchvision`, `keras`, `transformers`, `keras_cv` ...

Reply:
yeah I was thinking that this one could be better developed in the future as we expand the modeltypes (ModelInterface.NewModel) and the list of ModelTypes will basically be those